go-chaos/internal/pods.go

```diff
@@ -302,6 +303,35 @@ func (c K8Client) createPortForwardUrl(names []string) *url.URL {
 	return portForwardCreateURL
 }
 
+func (c K8Client) ExecuteCommandViaDebugContainer(podName string, containerName string, debugImage string, cmd []string) error {
+	pod, err := c.Clientset.CoreV1().Pods(c.GetCurrentNamespace()).Get(context.TODO(), podName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	name := "debug-" + rand.String(6)
+	debugContainer := v1.EphemeralContainer{
+		EphemeralContainerCommon: v1.EphemeralContainerCommon{
```

Comment:
Hm and this works? 🤔 Interesting. Could we use the same for zdb? I was always wondering why it didn't worked to access the data from ephemeral container. https://github.com/Zelldon/zdb/issues/332

Reply:
Yeah, I don't know what's going wrong in https://github.com/Zelldon/zdb/issues/332 but the debug container here has enough permissions to install the stress tool and run it at least.